src/acf.py

```diff
@@ -185,6 +208,12 @@ def ros_setup(self):
         self.create_service(SetFloat, '/ACF/set_payload', self.set_payload)
         self.create_service(SetFloat, '/ACF/set_f_zero', self.set_f_zero)
         self.create_service(SetDuration, '/ACF/set_t_ramp', self.set_t_ramp)
+        if self.frequency > 0:
+            self.create_timer(1 / self.frequency, self.timer_callback)
```

Comment:
```suggestion
            self.create_timer(1.0 / self.frequency, self.timer_callback)
```

it's always nicer to use appropriate types. Saves the interpreter from having to coerce them later, and shows intent better.

Reply:
Fixed in [6989bbf](https://github.com/sam-xl/ferrobotics_acf/pull/2/commits/6989bbfe18451b0168b58bb63db34b602eafa8e5)